src/content/docs/linter/index.mdx

```diff
@@ -4,180 +4,205 @@ description: How to use the Biome linter.
 
-```shell
-biome lint ./src
-```
-
-For more information about all the available options, check the [CLI page](/reference/cli#biome-lint)
+<PackageManagerBiomeCommand command="lint ./src" />
 
-## Rule pillars
-
-We believe that rules should be informative and explain to the user why a rule is triggered and tell the user what they should to do fix the error. A rule should follow these **pillars**:
-1. Explain to the user the error. Generally, this is the message of the diagnostic.
-2. Explain to the user **why** the error is triggered. Generally, this is implemented with an additional node.
-3. Tell the user what they should do. Generally, this is implemented using a code action. If a code action is not applicable a note should tell the user what they should do to fix the error.
+Note that if you pass a glob as a parameter, it will be expanded by your shell.
+The result of the expansion depends on your shell and its configuration.
+For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
 
-If you think a rule doesn't follow these pillars, please [file an issue](https://github.com/biomejs/biome/issues/new?assignees=&labels=S-To+triage&projects=&template=01_bug.yml&title=%F0%9F%90%9B+%3CTITLE%3E).
+```shell
+biome lint ./src/**/*.test.{js,ts}
+```
 
-## Code fixes
+For more information about all the available options, check the [CLI reference](/reference/cli#biome-lint).
 
-Lint rules may provide automatic code fixes. Biome distinguishes between two types of fixes.
 
-### Safe fixes
+## Rules
 
-Safe fixes are guaranteed to not change the semantic of your code. They can be applied without explicit review.
+The linter is organized into rules.
+A rule emit a diagnostic when it encounters a code that doesn't meet its requirements.
+For example, the [noDebugger](http://localhost:4321/linter/rules/no-debugger) rule reports the use of the `debugger` instruction in JavaScript code.
 
-To apply _safe fixes_, use `--apply`:
+A rule emits diagnostics with a `warn` or `error` severity.
+Diagnostics with an `error` severity cause the command to exit with a non-zero code,
+While diagnostics with a `warn` severity don't cause the command to fail.
+You can turn warnings into errors `--error-on-warnings` CLI option:
 
 ```shell
-biome check --apply ./src
+biome lint --error-on-warnings ./src
 ```
 
-### Unsafe fixes
+By default, the Biome linter only runs [**recommended rules**](/linter/rules/).
+Recommended rules emit diagnostics with the `error` severity.
 
-Unsafe fixes may change the semantic of your program. Therefore, it's advised to manually review the changes.
+Rules are organized into groups.
+For example, the `noDebugger` rule is part of the [`suspicious` group](/linter/rules#suspicious).
+A Rule from this group detects code that is likely to be incorrect or useless.
+The description of each group can be found on the [rules page](/linter/rules/).
```

Comment:
```suggestion
Our [rules page](/linter/rules/) explains each group in-depth.
```

There is no need to explain a group if there's a page already for that (we avoid repeating information and causing too much fragmentation).